en/docs/monitoring-and-insights/observability-overview.md

```diff
@@ -56,25 +56,38 @@ By default, the time range selected for the **Throughput & Latency** graphs is t
 A **bin** is a horizontal section of the graph for a particular period. A bin consists of the following items:
 
 - **Date/Time:** The timestamp for the log entries as they started to appear.
-- **Logs:**  List of log entries and their respective log count that occurred within the bins time frame. These log entries appear according to precedence. The error logs are listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted by the log count.
+- **Logs:**  List of log entries and their respective log count that occurred within the bin's time frame. These log entries appear according to precedence. The error logs are listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted by the log count.
 - **Error:** The number of HTTP errors that occurred at the selected time.
 - **TP:** Throughput of the requests at the selected time (req/s).
 - **Latency:** Latency of the request at the selected time (ms).
 - **CPU:** CPU usage at the selected time (millicores).
 - **Memory:** Memory usage at the selected time (MiB).
 
+## Logs
 
-## View and filter all logs
+The Choreo log panel offers a centralized view to access comprehensive runtime logs of the components you have deployed on Choreo, enhancing troubleshooting and analysis.
 
-The **Logs** panel includes all logs generated by the service. You can filter it based on time or based on text. The following figure shows how this might look:
+## Application logs
+
+![Logs panel](../assets/img/monitoring-and-insights/observability/view-application-logs.png){.cInlineImage-full}
+
+The Service components generates application logs. On the **Observability Overview** page you can click the **View Logs** icon on the header bar to view the application logs for the component as shown in the image above.
+
+!!! note
+    For REST API Proxy components, application logs will only be available if the `Log Message` policy is attached to the respective flows. 
+
+## Gateway logs
 
 ![Logs panel](../assets/img/monitoring-and-insights/observability/logs-panel.png){.cInlineImage-full}
 
-To download the log entries, click **Download**. As a result, the log entries are saved as a .zip file in the location you specify.
+The **Logs** panel includes all gateway access logs that the proxy service generates. Logs are currently **only available for REST API Proxy components**.  To view the access logs, icon on the header bar. This will be available for other components soon. 
```

Comment:
Might have to use the term pane here too.

Seems like we are missing some info in `To view the access logs, icon on the header bar.`
Should it be `click the icon on the header bar?

Reply:
done